docs/modules/game-object-creation/standard-usage.md

```diff
@@ -0,0 +1,42 @@
+<%(Toc)%>
+# Setup basic spawning
+
+## Setup your GameObject prefabs
+
+The [`GameObjectCreatorFromMetadata`]({{urlRoot}}/api/game-object-creation/game-object-creator-from-metadata) class is the default implementation for spawning GameObjects for your SpatialOS entities. This class expects your GameObject prefabs to be in a particular place in your project.
+
+The `GameObjectCreatorFromMetadata` uses the `entity_type` field in the `Metadata` component along with the worker instance's type to determine which prefab to spawn.
+
+<%(Callout message="You can set the `entity_type` field in the `Metadata` component when declaring your [EntityTemplate]({{urlRoot}}/reference/concepts/entity-templates.md)")%>
+
+For a given worker type `<worker-type>` it will look for a prefab at:
+
+```text
+Resources/Prefabs/<worker-type>/<Metadata.entity-type>
+```
+
+This leads to the following rules for a GameObject that will represent a SpatialOS entity:
+
+* The prefab must be named `<Metadata.entity_type>`.
+* For a specific worker, use the `Resources/Prefabs/<worker-type>` directory.
+* For a prefab that is common to any worker, use the `Resources/Prefabs/Common`.
+
+## Add the prerequisite systems
```

Comment:
```suggestion
## Add the GameObjectInitializationSystem system
```
maybe? as we only add one system anyway

Reply:
Well we don't ask users to add the system but call the helper - the helper can in the future add more than one system I guess. I really don't want to say `Call the helper` 😂

Reply:
"Set up your worker connector"?

I just don't really like the term "prerequisite systems"

Reply:
Sure